src/web/js/trove/chart-lib.js

```diff
@@ -401,14 +401,29 @@
 
   function pieChart(globalOptions, rawData) {
     const table = get(rawData, 'tab');
+    const default_colors = ['#3366CC', '#DC3912', '#FF9900', '#109618', '#990099',
```

Comment:
Leave a comment in the code saying you copied these from Google's default colors, and leave a link to there, so we can find them again.